MailCore/Cache/MainViewState.swift

```diff
@@ -17,30 +17,56 @@
  */
 
 import Foundation
+import InfomaniakCoreUI
+import InfomaniakDI
 import SwiftModalPresentation
 
 public protocol SelectedThreadOwnable {
     var selectedThread: Thread? { get set }
 }
 
 public class MainViewState: ObservableObject, SelectedThreadOwnable {
-    @ModalPublished public var composeMessageIntent: ComposeMessageIntent?
+    @ModalPublished public var composeMessageIntent: ComposeMessageIntent? {
```

Comment:
If we go beyond just a MVP maybe we should have a robust mechanism to observe changes on _all_ `ModalPublished`.